scheds/rust/scx_rusty/src/bpf/main.bpf.c

```diff
@@ -860,10 +913,19 @@ void BPF_STRUCT_OPS(rusty_stopping, struct task_struct *p, bool runnable)
 {
 	struct task_ctx *taskc;
 
-	if (fifo_sched)
+	if (!(taskc = lookup_task_ctx(p)))
 		return;
+		
+	s32 cpu = scx_bpf_task_cpu(p);
+	struct cpu_task_data *ctd;
 
-	if (!(taskc = lookup_task_ctx(p)))
+	ctd = MEMBER_VPTR(cpu_task_data, [cpu]);
+	if (ctd) {
+		ctd->has_running_task = false;
+	}
```

Comment:
nit: In kernel coding style, single-line conditionals, loops, etc don't use curly braces

Reply:
If there was one coding style I wish we could throw in the garbage and set on fire.